scripts/install_third_party_libs.py

```diff
@@ -211,6 +214,67 @@ def ensure_system_python_libraries_are_installed(package, version):
     install_backend_python_libs.pip_install_to_system(package, version)
 
 
+def rewrite_android_proto_files():
+    """Edit all android proto files."""
+    # Since there is no simple configuration for imports when using protobuf to
+    # generate Python files we need to manually fix the imports.
+    # See: https://github.com/protocolbuffers/protobuf/issues/1491
+    protobuf_dir = (
+        pathlib.Path(
+            os.path.join(
+                common.THIRD_PARTY_DIR,
+                'oppia-proto-api-introduce-proto-api-v1'))
+            .glob('**/*.proto'))
+    for p in protobuf_dir:
+        if p.suffix == '.proto':
+            # Remove package statement.
+            # Example - 'package org.oppia.proto.v1.api'.
```

Comment:
Use a colon after "Example" instead of a hyphen.

```
Example: 'package org ....
```

Reply:
Fixed